src/Bicep.Core/TypeSystem/DeployTimeConstantContainerVisitor.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Bicep.Core.Semantics;
+using Bicep.Core.Syntax;
+
+namespace Bicep.Core.TypeSystem
+{
+    public class DeployTimeConstantContainerVisitor : SyntaxVisitor
```

Comment:
Please could you add a comment explaining what a "Deploy Time Constant container" is - to help anyone looking at this for the first time?

Reply:
Done.